crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/controller/MeansAssessmentController.java

```diff
@@ -125,8 +125,7 @@ public ResponseEntity<ApiMeansAssessmentResponse> updateAssessment(@Parameter(de
                     schema = @Schema(implementation = ErrorDTO.class)
             )
     )
-    public ResponseEntity<ApiMeansAssessmentResponse> getOldAssessment(@PathVariable int financialAssessmentId,
-                                                                       @PathVariable String laaTransactionId) {
+    public ResponseEntity<ApiGetMeansAssessmentResponse> getOldAssessment(@PathVariable int financialAssessmentId, @PathVariable String laaTransactionId) {
```

Comment:
I don't think we should pass the LaaTransactionId as a path variable. This is meta information so shouldn't be part of the URL. Instead, this should be passed as a header.

Reply:
Fixed RequestHeader